sonar-csharp-plugin/pom.xml

```diff
@@ -156,7 +156,7 @@
           <pluginClass>org.sonar.plugins.csharp.CSharpPlugin</pluginClass>
           <skipDependenciesPackaging>true</skipDependenciesPackaging>
           <sonarLintSupported>false</sonarLintSupported>
-          <pluginApiMinVersion>${sonar.version}</pluginApiMinVersion> <!-- runtime minimal API version -->
+          <sonarQubeMinVersion>${sonarLTS.version}</sonarQubeMinVersion> <!-- runtime minimal API version -->
```

Comment:
I think the "API version" comment is misleading here. It may date back to the time when SQ version was equal to API version.